sway/desktop/render.c

```diff
@@ -1609,7 +1606,7 @@ void output_render(struct fx_render_context *ctx) {
 
 	if (!pixman_region32_not_empty(damage)) {
 		// Output isn't damaged but needs buffer swap
-		goto renderer_end;
```

Comment:
Why are we returning here? Don't we want to still render software cursors (which happens in render_end)?

Reply:
I don't think that the software cursor would even render due to the lack of any damage, so returning early would yield the same result. 

Upstream also does this